go.mod

```diff
@@ -48,10 +49,9 @@ require (
 	github.com/microsoft/azure-devops-go-api/azuredevops v1.0.0-b5
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/newrelic/newrelic-client-go v1.1.0
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/ginkgo/v2 v2.5.0
-	github.com/onsi/gomega v1.24.1
-	github.com/open-policy-agent/cert-controller v0.6.0
+	github.com/onsi/ginkgo/v2 v2.6.1
+	github.com/onsi/gomega v1.24.2
+	github.com/open-policy-agent/cert-controller v0.5.0
```

Comment:
Don't downgrade this package, we need latest version of it for the certificate generation

Reply:
Yes. I just saw the CI failure this morning. I have already made the changes